eclipse-collections/src/main/java/org/eclipse/collections/impl/factory/Multimaps.java

```diff
@@ -291,6 +294,62 @@ private ImmutableBagMultimapFactory()
                 return hashBagMultimap.toImmutable();
             }
         }
+
+        public static final class ImmutableSortedBagMultimapFactory
+        {
+           private ImmutableSortedBagMultimapFactory()
+           {
+           }
+
+           public <K, V> ImmutableSortedBagMultimap<K, V> of(Comparator<V> comparator)
```

Comment:
I see that the scope of Comparator was expanded for ImmutableSortedSetMultiple, but for the new implementation it is constricted 😄